compute/client_library/requirements.txt

```
@@ -1,5 +1,5 @@
 isort==6.0.0; python_version > "3.9"
 isort==5.13.2; python_version <= "3.8"
-black==24.8.0; python_version < "3.9"
-black==24.10.0; python_version >= "3.9"
+black==25.1.0; python_version < "3.9"
+black==25.1.0; python_version >= "3.9"
```

```suggestion
black==25.1.0
```

Not testing in 3.8 any more, no need to have this pinned